### PR TITLE
add kill-forward command.

### PR DESCRIPTION
**Pull request by mingyuan-xia**

### CODE REVIEW

File: lib/adb/client.ts [hunk 1]
```diff
@@ -21,6 +21,7 @@ import GetPropertiesCommand from './command/host-transport/getproperties';
 import InstallCommand from './command/host-transport/install';
 import IsInstalledCommand from './command/host-transport/isinstalled';
 import ListReversesCommand from './command/host-transport/listreverses';
+import KillForwardCommand from './command/host-serial/killforward';
```
Comment (koral--) [inline lib/adb/client.ts:24]: Why 2 words for the same thing: kill and remove?
I would rather stick to "remove".

Comment (mingyuan-xia) [reply lib/adb/client.ts:24]: the protocol says killforward, and for user, "remove" being more friendly

**Review by koral-- — APPROVED** [review]

**Comment by mingyuan-xia** [comment]
can we have a merge and a new release?

**Comment by koral--** [comment]
Could you fix merge conflicts?

**Comment by UrielCh** [comment]
Added to [@u4/adbkit 3.1.7](https://www.npmjs.com/package/@u4/adbkit)

**Comment by koral--** [comment]
@pcrepieux @denis99999 WDYT should we switch to external dependency in case of adbkit?

**Comment by pcrepieux** [comment]
> @pcrepieux @denis99999 WDYT should we switch to external dependency in case of adbkit?

I am afraid this raises a few more questions ;-)

- how to ensure the external dependency doesn't and also will not break stf ? (the conservative choice of using own's adbkit at least provides some sort of stability but maybe at the cost of a lack of evolution)
- @UrielCh's adbkit seems to do more things than just adb which I guess was the initial intent of adbkit. Wouldn't it overlap some other features that are in stf ? 
- However his version of adbkit seems to be more active. So, another question I have : why not contributing here ? (just wondering, I have no bias at all)

So basically, for now I'd stay with the initial version, but am open to carefully consider other options depending on the feedbacks I'll get.

**Comment by UrielCh** [comment]
Hi [pcrepieux](https://github.com/pcrepieux)
can you send me an email, in English or in French ?

and... yes my adbkit do much more than the original one, I have already built my own STF platform, in typescript + VueJS 3, currently working using STF service, but will sooner work with [Genymobile/scrcpy:](https://github.com/Genymobile/scrcpy).
